test/unit/auth/auth.spec.ts

```diff
@@ -1236,7 +1242,7 @@ AUTH_CONFIGS.forEach((testConfig) => {
       });
 
       it('should be rejected given invalid properties', () => {
-        return auth.createUser(null)
+        return auth.createUser(null as unknown as CreateRequest)
```

Comment:
Can we `null as any`?

Reply:
I have a slight preference towards `as any` since it's less things to type. But this ok too.